docs/dev/configuration.rst

```diff
@@ -0,0 +1,43 @@
+Configuration module
+=====================
+
+Front-end configuration
+------------
+
+It is possible to perform simple configuration of the MO frontend using the
+configuation file found in ``backend/mora/user_settings.json`` or in the file
+indicated in the env-variable USER_SETTINGS.
+
+This file could look something like this:
+
+    .. sourcecode:: json
+
+    {
+	"orgunit": {
```

Comment:
That unindent completely breaks rendering — you can inspect the generated output using `make -C docs` with your venv activated.